apps/dialtone-documentation/docs/design/sizing/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Sizing
+status: In progress
+description: A system to maintain consistent size and scale.
+---
+
+## Usage
+
+Use size units, when setting the scale of an element.
+
+When adding space around on in between elements use [Spacing](/design/spacing/) units.
+
+### Widths and Heights
+
+<div class="d-d-grid d-gg24 d-g-cols3 md:d-g-cols1">
+<dt-stack direction="column" gap="100">
+<p>
+
+For optimal responsiveness, we avoid using fixed `widths` or `heights` when defining element sizes. However, when setting a minimum dimensions, use `min-width` or `min-height`. This approach promotes flexibility and layout stability, key components of responsive design.
```

Comment:
I'm a bit confused by this paragraph, it talks about not setting fixed width / height but then below states dialtone components that DO have fixed width height. Are we saying that these components are exceptions to this rule?

It's also unclear whether we are promoting the usage of min-width / min-height. Are we promoting the use of min-width / min-height as much as possible for greater responsiveness? or are we saying that we should avoid using min-width / min-height unless you absolutely need to use it?